src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -288,7 +290,7 @@ public IActionResult Recover([FromBody]WalletRecoveryRequest request)
             {
                 Wallet wallet = this.walletManager.RecoverWallet(request.Password, request.Name, request.Mnemonic, request.CreationDate, passphrase: request.Passphrase);
 
-                this.SyncFromBestHeightForRecoveredWallets(request.CreationDate);
+                //this.SyncFromBestHeightForRecoveredWallets(request.CreationDate);
```

Comment:
If this is not needed any more you can remove this commented code completely :)

Reply:
Ok.